### DISCUSSION TITLE
Grafana incident and observability for on premisse intallations

### DISCUSSION THREAD

**Discussion by suikast42** [grafana/grafana#68361] [General]
I see the both featurs in play grafana page. 

Is that planed to port this features for on prem installations ? 

https://play.grafana.org/monitoring
https://play.grafana.org/a/grafana-incident-app/incidents

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
